pipelines/src/main/scala/dagr/pipelines/CreateUnmappedBamFromFastqPipeline.scala

```diff
@@ -53,7 +53,7 @@ class CreateUnmappedBamFromFastqPipeline(
   @Arg(doc="Path to a temporary directory.")                                      val tmp: Path,
   @Arg(flag="o", doc="The output prefix for files that are kept.")                val output: Path,
   @Arg(doc="Path to the unmapped BAM.  Use the output prefix if none is given. ") var unmappedBam: Option[PathToBam] = None
-) extends Pipeline {
+) extends Pipeline(Some(output.toAbsolutePath.getParent)) {
```

Comment:
I would be more tempted to add a method in `Io` like:
```scala:
def pathTo(path: String) : Path = Paths.get(path).toAbsolutePath
```
and use that everywhere we can instead of Paths.get.  What do you think?